src/main/java/com/impactupgrade/nucleus/service/segment/SynchronizationService.java

```diff
@@ -0,0 +1,8 @@
+package com.impactupgrade.nucleus.service.segment;
+
+import java.util.Calendar;
+
+public interface SynchronizationService extends SegmentService {
```

Comment:
Super picky: can we name this DataSyncService? Or something similar? Easier to type/remember. My tired brain has to remember how to spell Synchronization and it's harder to type...